README.md

```diff
@@ -37,6 +42,38 @@ Instead of specifying the packages directly, you can pass a file containing the
            package_file: tl_packages
 ```
 
+### Enabling updating cache using a workflow trigger
+
+One can inclue a cache referesh using GitHub's `workflow_dispatch` functionality:
+
+```yaml
+name: check
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      cache_version:
+        description: 'Cache version for install-texlive'
+        default: '0'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install TeX Live
+        uses: zauguin/install-texlive@v3
+        with:
+           package_file: tl_packages
+           cache_version: ${{ github.event.inputs.cache_version }}
+```
+
+## Caching
+
+The directory `~/texlive` is automatically cached.
+The cache is updated if it originiates from the previous day (or earlier).
+A cache refresh can also be forced by changing the `cache_version` parameter.
```

Comment:
This seems a bit misleading since it misses the difference between an update which happens after a day and a full reinstallation which happens only with `cache_version`. Maybe
```suggestion
The cache is updated if it originiates from the previous day (or earlier).
A full cache regeneration can be forced by changing the `cache_version` parameter.
```